lib/workload/stateless/stacks/bclconvert-interop-qc-pipeline-manager/scripts/trigger_bclconvert_interop_qc.sh

```diff
@@ -21,19 +21,19 @@ WORKFLOW_VERSION="1.3.1--1.21__20240410040204"
 #############
 
 # Inputs
-BCLCONVERT_REPORT_DIRECTORY_URI="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary_data/240229_A00130_0288_BH5HM2DSXC/202405302b817f5e/Reports/"
-INTEROP_DIRECTORY_URI="icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary_data/240229_A00130_0288_BH5HM2DSXC/202405302b817f5e/InterOp/"
-INSTRUMENT_RUN_ID="240229_A00130_0288_BH5HM2DSXC"
+BCLCONVERT_REPORT_DIRECTORY_URI="icav2://ea19a3f5-ec7c-4940-a474-c31cd91dbad4/primary_data/240424_A01052_0193_BH7JMMDRX5/20240606b794561d/Reports/"
```

Comment:
Out of curiosity: Why are some "Icav2://" path with project ID and others with project name?

Reply:
It doesn't matter, if it's not a uuid, wrapica converts from a project name to a project id. Mostly using project ids